client/src/views/TrackViewer/Layers.vue

```diff
@@ -195,16 +198,15 @@ export default defineComponent({
     annotationLayer.$on('annotationRightClicked', Clicked);
 
     editAnnotationLayer.$on('update:geojson',
-      (data: GeoJSON.Feature<GeoJSON.Polygon>, refresh: boolean) => {
+      (data: GeoJSON.Feature<GeoJSON.Polygon>) => {
         const track = props.trackMap.get(props.selectedTrackId.value);
         if (track) {
           const bounds = geojsonToBound(data);
           track.setFeature({
             frame: frameNumber.value,
             bounds,
           });
-          // We don't need to update the editing layer unless we create a new annotation
-          updateLayers(!refresh);
+          updateLayers();
```

Comment:
You don't need this line anymore.  `setFeature` will cause this function to call.

Reply:
Slipped my mind, the whole point of our discussion about bounds now triggering trackIds to update.